### PR TITLE
Don't display MODS or Dublin Core button for adminPolicy

### PR DESCRIPTION
**Pull request by jcoyne**
## Why was this change made? 🤔

This button shows the public MODS, which is a preview of what is published to PURL.  Since adminPolicies and agreements can't be published, don't show this preview.

Ref https://github.com/sul-dlss/dor-services-app/pull/3578#discussion_r812526754

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

⚡ ⚠ If this change updates the Argo UI, run all integration tests that use Argo and create a PR on that repo to fix anything this change breaks. ⚡

### CODE REVIEW

File: app/components/show/external_links_component.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module Show
+  # This draws links to external services for this object. Used on the sidebar of the item show page.
+  class ExternalLinksComponent < ViewComponent::Base
```
Comment (jcoyne) [inline app/components/show/external_links_component.rb:5]: This was just moved into this namespace, thus "everything changed" in the diff.

File: app/components/show/external_links_component.rb [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    # @param [SolrDocument] document
+    def initialize(document:)
+      @document = document
+    end
+
+    delegate :publishable?, to: :@document
```
Comment (jcoyne) [inline app/components/show/external_links_component.rb:11]: This is the only line I changed other than the namespace.

**Review by ndushay — COMMENTED** [review]
a) oh no - failing lint/tests

b) I'm a bit muddled from slack, but ... is this the solution Andrew wanted?

**Review by justinlittman — APPROVED** [review]
@ndushay based on my reading of the slack conversation, this seems like the correct solution.

**Comment by ndushay** [comment]
I thought that @andrewjbtw wanted to be able to use the MODS button for adminPolicy, since it has mods ... did that get resolved in a conversation somewhere that I missed?

**Comment by jcoyne** [comment]
I saw Andrew say, "In that case, we can remove the buttons from APOs and agreements. I would still like APOs to handle descriptive metadata internally through the bulk metadata upload function."  

Maybe I missed something?  Is there something else we should consider?

**Comment by ndushay** [comment]
I missed Andrew's comment in the other PR.  I just saw it.

**Pull request merged**